spec/cucumber/core/event_bus_spec.rb

```diff
@@ -0,0 +1,151 @@
+require "cucumber/core/event_bus"
+
+module Cucumber
+  module Core
+    module Events
+
+      class TestEvent < Core::Event.new(:some_attribute)
+      end
+
+      AnotherTestEvent = Core::Event.new
+
+      UnregisteredEvent = Core::Event.new
+    end
+
+    describe EventBus do
+      let(:bus) { EventBus.new(registry) }
```

Comment:
Fixed in 1862d85 (and also in Cucumber-Ruby, see https://github.com/cucumber/cucumber-ruby/pull/973/commits/b93a30b4ed7adb3ae5c6c66fb6e68b475d3ea9df).